src/ConstructiveSolidGeometry/LinePrimitives/Edge.jl

```diff
@@ -1,8 +1,29 @@
-struct Edge{T} <: AbstractLinePrimitive{T}
+struct Edge{T<:AbstractFloat} <: AbstractLinePrimitive{T}
```

Comment:
`<:AbstractFloat` is already in `abstract type AbstractGeometry{T <: AbstractFloat}` end.
Please remove it here.

Reply:
see discussion in #308

Reply:
Well `T` should be at least a `RealQuantity`... 
I see no setting in which it would make sense for `T` to be a `String`, for example...

Reply:
Julia: Never restrict types if not necessary. 
This allows for different packages to work together without knowing each other. 
The only reason to restrict types is for dispatching in general. 

I could live with `Number` which is supposed to be the supertype of all types which have something to do with units. 
It's also the supertype of a Quantity. 
Another example would be if people use the CSG objects in automatic differentiation. 
Those packages have special types for Numbers, e.g. `ForwardDiff.Dual`. 

But since there is no reason to restrict, as we do not get any benefit from it, we should not restrict at all.